dashboard/src/modules/components/TableComponent/index.jsx

```diff
@@ -235,21 +235,21 @@ const TableWithFavorite = () => {
                   )}
                 </Tbody>
               </TableComposable>
+              <TablePagination
+                numberOfRows={
+                  isSelected === "datasetListButton"
+                    ? tableData.length
+                    : favoriteRepoNames.length
+                }
+                page={page}
+                setPage={setPage}
+                perPage={perPage}
+                setPerPage={setPerPage}
+              />
             </InnerScrollContainer>
           </OuterScrollContainer>
```

Comment:
Why do we have both an `InnerScrollContainer` and an `OuterScrollContainer`?  I don't see anything inside the outer container other than the inner container...what am I missing?  (If there is effectively no difference between the two, then are we sure we want them both?  If there _is_ a difference, are we sure we want the `TablePagination` inside the inner one, instead of inside just the outer one after the inner one?)

Reply:
It is the design feature of Pattenfly to maintain proper sticky behaviour of sticky column header of Patternfly table.

So, the table must be wrapped with `OuterScrollContainer` and `InnerScrollContainer`.